sdk/connectedvmwarevsphere/Azure.ResourceManager.ConnectedVMwarevSphere/src/Generated/ResourcePoolData.cs

```diff
@@ -63,7 +63,7 @@ public ResourcePoolData(AzureLocation location) : base(location)
         /// <param name="customResourceName"> Gets the name of the corresponding resource in Kubernetes. </param>
         /// <param name="statuses"> The resource status information. </param>
         /// <param name="provisioningState"> Gets or sets the provisioning state. </param>
-        internal ResourcePoolData(ResourceIdentifier id, string name, ResourceType type, IDictionary<string, string> tags, AzureLocation location, ExtendedLocation extendedLocation, SystemData systemData, string kind, string uuid, string vCenterId, string moRefId, string inventoryItemId, string moName, string cpuSharesLevel, long? cpuReservationMHz, long? cpuLimitMHz, string memSharesLevel, long? memReservationMB, long? memLimitMB, string customResourceName, IReadOnlyList<ResourceStatus> statuses, string provisioningState) : base(id, name, type, tags, location)
```

Comment:
Please updated the sdk code to latest version

Reply:
You mean, I should update with this: internal ResourcePoolData(ResourceIdentifier id, string name, ResourceType type, IDictionary<string, string> tags, AzureLocation location, ExtendedLocation extendedLocation, SystemData systemData, string kind, string uuid, string vCenterId, string moRefId, string inventoryItemId, string moName, string cpuSharesLevel, long? cpuReservationMHz, long? cpuLimitMHz, string memSharesLevel, long? memReservationMB, long? memLimitMB, string customResourceName, IReadOnlyList<ResourceStatus> statuses, string provisioningState) : base(id, name, type, tags, location)

Reply:
Means I should update with where we are using  "AzureLocation"  param instead of "Location"?

Reply:
Yes, you can just sync up the code from main branch

Reply:
Done.